### ISSUE TITLE
Move addin to cake-contrib organization

### ISSUE THREAD

**Issue by AdmiringWorm** [cake-contrib/Cake.Transifex#5]
As the title suggest, start migrating Cake.Transifex to the cake-contrib team.

**LIST FOR STEPS TO DO FOLLOWS LATER**

Would like some feedback on what is needed to move this addin to the cake-contrib,
both for the process and for any necessary changes I need to do with the code.
(Main reason for the move is discover-ability, and possibility for bigger chance of suggestions on how to improve the addin).

/cc @agc93 @devlead @gep13 @patriksvensson

**Comment by gep13** [comment]
@AdmiringWorm would be more than happy to have you onboard.

At the minute, there are no hard and fast requirements to getting your repository into the cake-contrib Organisation, we are simply happy to welcome you into it.  There are however some suggested best practices, which are documented here:

https://github.com/cake-contrib/Home/blob/master/Guidelines.md

As I said, these are not hard and fast, and don't need to be done straight away, these can be implemented over time.

**Comment by gep13** [comment]
In terms of the actual transfer, the easiest way I have found to do this would be to have you transfer the repo to my individual GitHub user account, and from there, I can move it into the Organisation.  From there, I would then set up the necessary permissions and AppVeyor Project etc, that is required, giving you full access to both.

**Comment by AdmiringWorm** [comment]
Sounds great, don't want to move this repo right now though.
I'm planning to use this extension to test out the travis portion of https://github.com/cake-contrib/Cake.Recipe/pull/111 (the .NET Core support), and since I got everything already set up here (only missing the Cake.Recipe build file) it would be easier to test it out now before moving.

BTW, in the [Cake Documentation Guidelines](http://cakebuild.net/docs/contributing/documentation) mentioned in the linked guidelines there look to be something wrong.
Basically their missing the tags for the comment guidelines on `Classes` and `Methods`.

**Comment by gep13** [comment]
@AdmiringWorm not a problem at all, makes total sense.

Hmm, that is strange.

@daveaglick any thoughts on what is going on here:

![image](https://cloud.githubusercontent.com/assets/1271146/26765029/0fde62ee-496b-11e7-96f4-15feb6355aac.png)
